lib/providers/okta.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// Load modules
+const Joi = require('joi');
+const Hoek = require('hoek');
+
+// Declare internals
+
+const internals = {
+    schema: Joi.object({
+        uri: Joi.string().uri().required()
+    }).required()
+};
+
+exports = module.exports = function (options) {
+
+    const results = Joi.validate(options, internals.schema);
+    Hoek.assert(!results.error, results.error);
+    const settings = results.value;
```

Comment:
This is following a very similar pattern as https://github.com/hapijs/bell/blob/master/lib/providers/auth0.js#L12. 

It was my understanding that line 17 using `Joi.validate` would ensure folks pass in the expected schema.